src/userbase-js/src/ws.js

```diff
@@ -186,18 +186,30 @@ class Connection {
 
               if (!database.dbKey) throw new Error('Missing db key')
 
+              const databaseWritersMap = message.dbWriters && message.dbWriters.reduce((map, { userId, username }) => map.set(userId, username), new Map)
```

Comment:
Maybe this should be a property on the database object, because I could use it when building a new bundle.